src/starknet-wrappers.ts

```diff
@@ -23,11 +29,20 @@ interface CompileWrapperOptions {
 }
 
 interface Cairo1CompilerOptions {
+    path: string;
+    contractPath?: string;
+    output?: string;
```

Comment:
How are `contractPath` and `output` optional?

Reply:
While we always provide `output` for user `contractPath` is indeed optional because you can pass contract path in `path` (what we do rn).
Regarding output I guess we can continue [discussion in this comment](https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/345/files/f612422733a084bb595c587be8fc9579abb00a97#r1179250554). I don't really have any strong opinion on this

Reply:
Removed contractPath